s/shiny-server/shiny-server_ubi_9.3.sh

```diff
@@ -0,0 +1,120 @@
+#!/bin/bash -ex
+# -----------------------------------------------------------------------------
+#
+# Package       : shiny-server
+# Version       : v1.5.23.1030
+# Source repo   : https://github.com/rstudio/shiny-server
+# Tested on     : UBI:9.3
+# Language      : Java/C++
+# Travis-Check  : True
+# Script License: Apache License, Version 2 or later
+# Maintainer    : Guarav.Bankar@ibm.com
+#
+# Disclaimer: This script has been tested in user mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+PACKAGE_URL=https://github.com/rstudio/shiny-server
+PACKAGE_NAME=shiny-server
+PACKAGE_VERSION=${1:-v1.5.23.1030}
+export ARCH=$(uname -m)
+
+# Install required packages
+echo "Installing required dependencies..."
+yum install -y gcc gcc-c++ gcc-gfortran git wget xz cmake make yum-utils python3 python3-devel llvm sudo npm nodejs
+
+git clone $PACKAGE_URL /root/$PACKAGE_NAME
+cd /root/$PACKAGE_NAME
+git checkout $PACKAGE_VERSION
+
+# Node installation via NVM
+export NODE_VERSION=${NODE_VERSION:-20}
+echo "Installing Node.js version $NODE_VERSION..."
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.0/install.sh | bash
+source "$HOME/.bashrc"
+nvm install "$NODE_VERSION" >/dev/null
+nvm use $NODE_VERSION
+
+# Update PATH to include local binaries and Python
+echo "Updating PATH..."
+export PATH=$(pwd)/../bin:$PATH
+PYTHON=$(which python3)
+export PATH=$PYTHON:$PATH
+
+# Build the Shiny Server
+cmake -DCMAKE_INSTALL_PREFIX=/usr/local -DPYTHON="$PYTHON"
+
+if ! make ; then
+    echo "------------------$PACKAGE_NAME:install_fails-------------------------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_Fails"
+    exit 1
+fi
+
+mkdir build
+npm ci --omit-dev
+
+# Install Shiny server
+if ! make install; then
+    echo "------------------$PACKAGE_NAME:install_success_but_test_fails---------------------"
+    echo "$PACKAGE_URL $PACKAGE_NAME"
+    echo "$PACKAGE_NAME  |  $PACKAGE_URL | $PACKAGE_VERSION | GitHub | Fail |  Install_success_but_test_Fails"
+    exit 2
+fi
+
+# Create a tarball of the installed Shiny server
+cd /root
```

Comment:
@gauravpbankar13 Lets not keep this path hardcoded. .. you can use $HOME variable or set certain variable at start of script to set the path. use it for relative movement,

Reply:
Ok. Initially I was having variable for the same but @vinodk99  ask me to have it hardcoded value.
Will update it.

Reply:
Hi @nikhil-kalbande 
I have added require changes.
Can you please review it?